frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'global/state';
+
+import { OpenEdXInstanceDeploymentStatusStatusEnum as DeploymentStatus } from 'ocim-client';
+
+import { Col, Row } from 'react-bootstrap';
+
+import { getDeploymentsAlerts } from 'console/actions';
+import { DeploymentAlertModel } from 'console/models';
+import { WrappedMessage } from 'utils/intl';
+
+import messages from './displayMessages';
+import { ConsolePage } from '../ConsolePage';
+
+import './styles.scss';
+
+interface ActionProps {
+  getAlerts: Function;
+}
+interface Props extends ActionProps {
+  alerts: Array<DeploymentAlertModel>;
+}
+
+interface AlertProps {
+  alert: DeploymentAlertModel;
+}
+
+const iconClassMap = {
+  [DeploymentStatus.Healthy]: 'fa-check-circle',
+  [DeploymentStatus.ChangesPending]: 'fa-check-circle',
+  [DeploymentStatus.Provisioning]: 'fa-info-circle',
+  [DeploymentStatus.Preparing]: 'fa-info-circle',
+  [DeploymentStatus.Unhealthy]: 'fa-times-circle',
+  [DeploymentStatus.Offline]: 'fa-stop-circle'
+};
+
+const statusClassMap = {
+  [DeploymentStatus.Healthy]: 'healthy',
+  [DeploymentStatus.ChangesPending]: 'changes_pending',
+  [DeploymentStatus.Provisioning]: 'provisioning',
+  [DeploymentStatus.Preparing]: 'preparing',
+  [DeploymentStatus.Unhealthy]: 'unhealthy',
+  [DeploymentStatus.Offline]: 'offline'
+};
+
+const Alert: React.FC<AlertProps> = (props: AlertProps) => {
+  const [detailsCollapsed, setDetailsCollapsed] = React.useState(true);
+
+  const deploymentStatus = props.alert.status;
+
+  const caretClass = detailsCollapsed ? 'fa-caret-down' : 'fa-caret-up';
+  const collapsedClass = detailsCollapsed ? 'collapsed' : '';
+  const iconClass = iconClassMap[deploymentStatus];
+  const statusClass = statusClassMap[deploymentStatus];
+
+  const month = props.alert.date.toLocaleString('default', { month: 'long' });
```

Comment:
I believe that this would produce exactly what we want?
```js
const date = props.alert.date.toLocaleDateString('default', {month: 'long', year: 'numeric', day: 'numeric'})
```